ovos_plugin_manager/utils/__init__.py

```diff
@@ -18,24 +18,43 @@
 
 class PluginTypes(str, Enum):
     PHAL = "ovos.plugin.phal"
+    PHAL_CONFIG = "ovos.plugin.phal.config"
```

Comment:
Would it be more readable to use a different enum for Config? `PluginTypes.PHAL`/`PluginConfig.PHAL`